modules/dashboard/README.md

```diff
@@ -0,0 +1,33 @@
+# Dashboard
+
+## Purpose
+
+The Dashboard serves as a landing page for all users logging into LORIS.  It provides introductory information and a primary quick-access interface for user workflows, including outstanding tasks and notifications of recent activity/additions in LORIS.  
+
+The Dashboard also provides at-a-glance live visual summaries of the data stored in LORIS, and dynamically plots data breakdowns that can be used in reporting. 
+
+## Intended Users
+
+All LORIS users can view the dashboard.
+
+## Scope
+
+The Dashboard provides entrypoints to various workflows, but does not provide any tools for directly modifying data in the dashboard. All users, regardless of site or project affiliation, can see data summaries and Tasks/Notifications across sites and projects. 
+
+The Study Progression chart views at bottom may or may not count `excluded` candidates, and this should be clearly verified and documented/presented for the use of your LORIS users. 
+
+## Permissions
+
+Components of the Dashboard are governed by permissions related to module-specific privileges. For example, the _Document Repository_ notifications panel is displayed if a user has been granted either the `document_repository_view` or `document_repository_delete` permission. 
```

Comment:
Are these the only permissions? If not, the rest should be enumerated here.

Reply:
The permissions are module-specific and are defined within each module's codebase, not by the dashboard.  Listing them here would mean reduplicating that information and requiring an update here if that module's permissions change.

Reply:
@driusan Is this now up to the modules since you made the widget changes?

Reply:
My comment was from before modules were adding the widgets themselves when the dashboard was handling everything itself, it's no longer relevant.